docs/abstract.md

```diff
@@ -1,14 +1,9 @@
 # Abstract
 
-## Vision
-Mit Arkivar soll eine plattformunabhängige Mobile-App für die Kunden von Imagic und PIC System erstellt werden. Sie erlaubt digitale Medien auf einfache Art im Imagic IMS Archiv abzuspeichern. Durch eine hohe Softwarequalität sollen zukünftige Erweiterungen einfach realisiert werden können.
+Ziel der Arbeit war, eine plattformunabhängige Mobile-App zu entwickeln, die Kunden von Imagic Bildverarbeitung AG und PIC Systems AG erlaubt, digitale Medien auf einfache Art im Imagic IMS Archiv abzuspeichern. Dabei soll eine hohe Softwarequalität angestrebt werden, sodass zukünftige Erweiterungen einfach realisiert werden können. Unsere persönliche Ziele waren Cross-Platform Frameworks und Continuous Deployment kennen zu lernen.
 
-Persönliche Ziele sind es Cross-Platform Frameworks und Continuous Deployment kennen zu lernen.
+Wir verwendeten das Angular und Ionic Framework und entschlossen uns die Applikationslogik im Functional Reactive Programming Paradigma zu implementieren.
 
-## Umsetzung 
-Mithilfe einer gemeinsamen Codebasis wurde eine Android- und iOS-App sowie eine Browserapplikation erstellt. Die Software kann mit sämtlichen kundenspezifischen Datenmodellen umgehen.
+Als Software Prozess Framework haben wir Scrum mit Extreme Programming ausgewählt. Durch Pair Programming, Code-Reviews, automatisiertes Testing und statische Codeanalyse wurde die Softwarequalität sichergestellt. In einem nahezu vollständig automatisierten Release Prozess wurden aus der selben Codebasis regelmässig neue Versionen für Android und iOS freigegeben.
 
-Durch die Verwendung des Angular und Ionic Frameworks entschlossen wir uns die Applikationslogik im Functional Reactive Programming Paradigma zu erstellen. Als Software Prozess Framework haben wir Scrum mit Extreme Programming ausgewählt. Durch Pair Programming, Code-Reviews, automatisiertes Testing und statischer Codeanalyse wurde die Softwarequalität sichergestellt. Dank einem automatisierten Deployment Prozess gelangen neue Releases rasch zum Kunden.
-
-## Fazit
-Das Ziel von unterschiedlichen mobilen Geräten Bilder hochzuladen wurde erreicht. Auch den aufkommenden Wunsch einer browserbasierten Version konnten wir erfüllen. Zusätzlich motivierend war, dass mit Continuous Deployment neue Features sofort vom Kunden genutzt werden können.
+Mit Arkivar steht nun eine Software zur Verfügung, die in der Lage ist, von unterschiedlichen mobilen Geräten Bilder hochzuladen und dabei mit sämtlichen kundenspezifischen Datenmodellen umgehen kann. Auch den aufkommenden Wunsch nach einer browserbasierten Version konnten wir umsetzen. Zusätzlich motivierend war, dass wir mithilfe von Continuous Deployment dem Kunden eine sofortige Nutzung neuer Features ermöglichen konnten.
```

Comment:
...Features ermöglichen konnten. 
I'm not sure, but this sounds little bit confusing. BUT I'm not sure :)